### DISCUSSION TITLE
Are there any suggested versions for a production environment?

### DISCUSSION THREAD

**Discussion by remote-tty1** [nilsteampassnet/TeamPass#3680] [Q&A]
Hi!
I would like to know if there are any stable versions that i could use to deploy teampass in a production environment
and also if there is some tag that I could use to find out if there are upgrade releases for the stable channel

Thanks in advance

**Comment by nilsteampassnet** [comment]
Thank for your question.
You should use the latest release.
Teampass is a living project. You will need to regularly update to have usage of latest fixes and improvements.
I'm trying to release a version every week.
